dev/TeachingTip/TeachingTip.cpp

```diff
@@ -876,6 +988,25 @@ void TeachingTip::OnPopupClosed(const winrt::IInspectable&, const winrt::IInspec
     auto myArgs = winrt::make_self<TeachingTipClosedEventArgs>();
     myArgs->Reason(m_lastCloseReason);
     m_closedEventSource(*this, *myArgs);
+
+    //If we were closed by the close button and we have tracked a previously focused element because F6 was used
+    //To give the tip focus, then we return focus when the popup closes.
+    if (m_lastCloseReason == winrt::TeachingTipCloseReason::CloseButton)
+    {
+        if (auto&& perviouslyFocusedElement = m_previouslyFocusedElement.get())
+        {
+            perviouslyFocusedElement.Focus(winrt::FocusState::Keyboard);
+        }
+    }
+    m_previouslyFocusedElement.set(nullptr);
+
+    if (auto peer = winrt::FrameworkElementAutomationPeer::FromElement(*this))
+    {
+        if (auto teachingTipPeer = peer.as<winrt::TeachingTipAutomationPeer>())
```

Comment:
>peer.as<winrt::TeachingTipAutomationPeer>() [](start = 35, length = 43)

If you think this might fail then use try_as. Also can be combined with previous as .as and .try_as (finally) defend against the thing being null.